src/stores/AccountController.ts

```diff
@@ -140,6 +140,20 @@ export const getAccount = async () => {
   });
 };
 
+export const getUserInfo = async () => {
+  return await useService(async ({ accountService }) => {
+    try {
+      const user = await accountService.getAccountInfo({ shouldFetchData: true });
+      if (user) {
+        useAccountStore.setState({ user });
+        return user;
```

Comment:
Maybe don't return the user from this method? It's assumed that the function returns the user data when called, but this will never be the case for the Cleeng integration.

The return data is not being used as of now, but we may make this mistake in the future.

Reply:
Yeah, there was an issue if the data was not returned, the isLoading state was not working properly or something, cant quite remember :D.
Thanks @ChristiaanScheermeijer  for the input, but this is on hold for now, will convert it to draft.